test/games/strategy/triplea/ui/TestRoute.java

```diff
@@ -97,6 +96,7 @@ private static int randomInt(int max, int min) {
     return (int) ((Math.random() * (max - min)) + min);
   }
 
+  @SuppressWarnings("unused")
```

Comment:
I have to rework the tests anyway. Removing this once the tests are working again...